0xProject/0x-monorepo
# [instant] Rollbar integration

## Description

Report Instant errors to Rollbar

⚠️ There are a lot of commits in here, would like to squash to merge ⚠️ 

⚠️ Before Merge: ⚠️ 
- [ ] Figure out how to publish sourcemaps on npm publish
- [ ] Fix `test-publish` error -- not sure what's happening here, but may be related to the webpack config change?

## Testing instructions

- Add change to a react component to make it easier to trigger an error.  (i.e. Typing in a magic number to the input makes an error trigger)
- Deploy to dogfood
- Trigger error and see it reported to rollbar w/ source maps and the correct environment set

## Types of changes









## Checklist:
